src/_pyinstaller_hooks_contrib/hooks/stdhooks/hook-accessible_output2.py

```diff
@@ -1,19 +1,8 @@
-# ------------------------------------------------------------------
-# Copyright (c) 2020 PyInstaller Development Team.
-#
-# This file is distributed under the terms of the GNU General Public
-# License (version 2.0 or later).
-#
-# The full license is available in LICENSE.GPL.txt, distributed with
-# this software.
-#
-# SPDX-License-Identifier: GPL-2.0-or-later
-# ------------------------------------------------------------------
-
```

Comment:
Don't remove the license. Without it anyone could add their own copyright to claim it as theirs, then sue PyInstaller and every PyInstaller user for intellectual property breach.

Reply:
Sorry. Wasn't my intent. I fixed the hooks some time ago, for a private project, and just forgot the license